CHANGELOG.md

```diff
@@ -4,6 +4,72 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/) and [Keep a CHANGELOG](https://keepachangelog.com/).
 
+### [2.0.0] - 2019-12-19
+
+#### Added
+* PHPCS: New `Yoast.NamingConventions.ObjectNameDepth` sniff.
+    - For objects _declared within a namespace_, this sniff verifies that an object name consist of maximum three words separated by underscores.
+    - For objects which are part of a unit test suite, a `_Test`, `_Mock` or `_Double` suffix at the end of the object name will be disregarded for the purposes of the word count.
+    - The sniff has two configurable properties `max_words` (error) and `recommended_max_words` (warning). The default for both is `3`.
+* PHPCS: New `Yoast.NamingConventions.NamespaceName` sniff.
+    This sniff verifies that:
+    - Namespace names consist of a maximum of three levels (excluding the plugin specific prefix) and recommends for the name to be maximum two levels deep.
+        For unit test files, `Tests\(Doubles\)` directly after the prefix will be ignored when determining the level depth.
+    - The levels in the namespace name directly translate to the directory path to the file.
+    - The sniff has four configurable properties:
+        - `max_levels` (error) and `recommended_max_levels` (warning) which are by default set to `3` and `2` respectively.
+        - `src_directory` to indicate the project root(s) for the path to name translation when the the project root is not the repo root directory.
```

Comment:
```suggestion
        - `src_directory` to indicate the project root(s) for the path-to-name translation when the project root is not the repo root directory.
```

Reply:
Fixing these up as we speak. Want me to adjust the release date as well ?